packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -20,28 +20,54 @@
       >
         <div
           data-cy="specs-list-row-git-info"
-          class="col-span-2"
         >
           <slot name="git-info" />
         </div>
-        <div>
-          <slot name="latest-runs" />
-        </div>
-        <div class="hidden md:col-span-2 md:block">
-          <slot name="average-duration" />
-        </div>
+        <SpecsListHoverCell
+          data-cy="specs-list-row-latest-runs"
+          :is-hover-disabled="isProjectConnected"
+        >
+          <template #content>
+            <slot name="latest-runs" />
+          </template>
+          <template #hover>
+            <slot
+              name="connect-button"
+              utmMedium="Specs Latest Runs Empty State"
```

Comment:
Are we testing that these UTM medium values get used?

Reply:
Looks like we would want to stub/spy on the mutation called from Auth.vue. Looking around now to see if we have similar coverage.

Reply:
Added test to make sure correct `utmMedium` parameter  is passed to slot differently depending on which row you hover over.